src/app/shared/models/asset.model.ts

```diff
@@ -1,12 +1,12 @@
 import { AssetCategory } from "./asset-category.model"
 import { Media } from "./media.model";
 export class AssetModel {
-    identifier: number
+    identifier: string
     category: AssetCategory;
     name: string;
     description: string;
     platform: string;
-    platform_resource_identifier: number;
+    platform_resource_identifier: string;
```

Comment:
This one hasn't changed and has always been a string. It just happens that some of the platforms (e.g., openml) use integers -- but even then on AIoD they are strings.